config/config.php

```diff
@@ -569,6 +569,38 @@
 	],
 
 	'beta' => [
+		'25' => [
+			'100' => [
+				'latest' => '25.0.0 beta 1',
```

Comment:
"beta 1" or "beta1" ? do we have guidelines ? past record ?

Reply:
It's taken from the server version.php 
https://github.com/nextcloud/server/blame/cd95fce105fe5f0e71b1bcac7685464f936b9749/version.php#L36

Reply:
It's `25.0.0 RC1`, `25.0.0 dev`, `25.0.0 beta 1` and  `25.0.0`